packages/kbn-config-schema/src/types/object_type.test.ts

```diff
@@ -199,3 +199,15 @@ test('includes namespace in failure when wrong value type', () => {
 
   expect(() => type.validate(value, {}, 'foo-namespace')).toThrowErrorMatchingSnapshot();
 });
+
+test('reach returns a valid schema component', () => {
+  const type = schema.object({
+    foo: schema.boolean(),
+  });
+
+  const value = false;
+  expect(() => type.validateKey('foo', value)).toBeTruthy();
```

Comment:
I don't think this works, I'm pretty sure expect is just saying that functions are truthy values 😬

Reply:
oh lol :)